src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,31 @@
+import React from "react";
+import Image from "next/image";
+import Graves from "public/team/graves.png";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div className="bg-gradient-to-r from-hlg-blue-300 to-hlg-black">
+      <div className="flex ">
+        <Image
+          src={Graves} // Path to your image inside the 'public' directory
+          alt="Graves Image"
+        />
+
+        <div className="mt-10 pl-10">
+          <div className="bg-gradient-to-r from-hlg-blue-100 to-hlg-blue-200 bg-clip-text text-transparent">
+            <div className="font-bold leading-tight text-5xl -mb-3 m-0 p-0">
+              <p>Matthew &quot;GetTurtled&quot;</p>
```

Comment:
the words r really far apart because of \<p\> default styles. remove default styles by adding m-0 p-0 for all \<p\> tags